examples/stocks/lib/stock_symbol_viewer.dart

```diff
@@ -15,6 +15,7 @@ class _StockSymbolView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    assert(stock != null);
     final String lastSale = "\$${stock.lastSale.toStringAsFixed(2)}";
```

Comment:
We should probably use the NumberFormat class here, just to demo how to format prices.